app/src/pages/trace/SpanDetails.tsx

```diff
@@ -1449,56 +1456,144 @@ function LLMMessage({ message }: { message: AttributeMessage }) {
           ) : null}
         </ErrorBoundary>
         <Flex direction="column" alignItems="start">
-          {messageContent ? (
-            <View padding="size-200">
-              <ConnectedMarkdownBlock>{messageContent}</ConnectedMarkdownBlock>
-            </View>
-          ) : null}
-          {toolCalls.length > 0
-            ? toolCalls.map((toolCall, idx) => {
-                const parsedArguments = safelyParseJSON(
-                  toolCall?.function?.arguments as string
-                );
+          <DisclosureGroup
```

Comment:
The actual rendered content for all of these content types remains the same, I just wrapped them in disclosures so the indentation makes this look more invasive than it is